pkgs/by-name/cl/clipse/package.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "clipse";
+  version = "0.0.6";
+
+  src = fetchFromGitHub {
+    owner = "savedra1";
+    repo = "clipse";
+    rev = "v${version}";
+    hash = "sha256-DLvYTPlLkp98zCzmbeL68B7mHl7RY3ee9rL30vYm5Ow=";
+  };
+
+  vendorHash = "sha256-GIUEx4h3xvLySjBAQKajby2cdH8ioHkv8aPskHN0V+w=";
+
+  meta = {
+    description = "A useful clipboard manager TUI for Unix";
```

```suggestion
    description = "Useful clipboard manager TUI for Unix";
```
see contributing guide

---

For my own future reference see section [**Meta attributes** in `pkgs/README.md`](https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes)

---

Thanks both - have updated your suggestions now @SuperSandro2000 :+1: